views/tracks.hbs

```diff
@@ -0,0 +1,5 @@
+{{#each tracks}}
+<h1>{{name}}</h1>
+<h1>{{preview_url}}</h1>
+<iframe src="https://open.spotify.com/embed?uri={{uri}}" width="300" height="380" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>
```

Comment:
This is cool! I remember you had an issue with audio controls. It worked for me like so: 
 ```
       <div>
            <audio controls>
            <source src="{{this.preview_url}}" type="audio/mpeg">
            Your browser does not support the audio element.
            </audio>
        </div>
```